### ISSUE TITLE
[Netherstorm] Missing quest and incorrect progression mark display

### ISSUE THREAD

**Issue by mara1987** [ccshiro/cc-buglist#524]
Hello again,

I decided to test some quest in Netherstorm, in particular the repeatable one in Area 52 for Consortium reputation:

http://www.wowhead.com/quest=10262/a-heap-of-ethereals
and
http://www.wowhead.com/quest=10308/another-heap-of-ethereals

The first issue i found was an incorrect progression mark display, in fact i should have get a grey question mark, but instead: 
![incorrect mark display](https://cloud.githubusercontent.com/assets/10872722/7257013/d33c9b06-e852-11e4-98d9-865544efdb0f.JPG)

Then after turning the quest in (Notice that the yellow question mark is not referred to the warp Nesaad, I had yet to complete it):
![missing repeatable quest](https://cloud.githubusercontent.com/assets/10872722/7257040/05b01e14-e853-11e4-81c8-e3b0e93ee956.JPG)

Then, after turning (http://www.wowhead.com/quest=10205/warp-raider-nesaad) quest:
![missing repeatable quest 2](https://cloud.githubusercontent.com/assets/10872722/7257088/6b2b3742-e853-11e4-85fc-9ec48022fa0e.JPG)

So, in conclusion. You don't get the repeatable quest (blue question mark display) but you have a "completed evergreen" displayed.

Drop rate of the quest item (http://www.wowhead.com/item=29209/zaxxis-insignia) is correct.

**Comment by mara1987** [comment]
After logging out and in again the yellow question mark seemed to be gone for good.
![after logout](https://cloud.githubusercontent.com/assets/10872722/7257322/28ad9b6a-e855-11e4-8b1f-521ba9cad344.JPG)

**Comment by mara1987** [comment]
Ok, i managed to get Friendly reputation before shutdown. Now the NPC gives the correct repeatable quest and displays the correct question mark:

![correction](https://cloud.githubusercontent.com/assets/10872722/7259017/a4060d8c-e860-11e4-85b6-7c6008565a31.JPG)

So, in the end that was only a reputation issue. The wrong progression marker issue remains though.

**Comment by ccshiro** [comment]
@ccnim will check the reputation stuff.

As for the yellow question mark, we cannot reproduce that. Needs some more testing and info what goes on.

**Comment by mara1987** [comment]
I trusted the screens to be enough...

**Comment by ccnim** [comment]
Reputation part is done, you needed to be friendly to be able to accept "A Heap of Ethereals". As for the question mark problem, we are unable to reproduce it.

**Comment by ccshiro** [comment]
Moved to needs testing; maybe someone can figure out how to reproduce the bug.

**Comment by mara1987** [comment]
I confirm that the reputation issue is resolved. Now if you are not Friendly with Consortium you can't pick up the quest.

**Comment by lynxzor** [comment]
Nothing fancy to reproduce just turned in the quest Assisting the Consortium and upon taking Consortium Crystal Collection the yellow question mark still remains

![wowscrnshot_060715_153518](https://cloud.githubusercontent.com/assets/795881/8022517/12437390-0d2b-11e5-93d6-7e2f6d52bfd3.jpg)

Then after completing Consortium Crystal Collection

![wowscrnshot_060715_154016](https://cloud.githubusercontent.com/assets/795881/8022526/9409e508-0d2b-11e5-97f7-fa200f6012c1.jpg)

**Comment by mara1987** [comment]
I have tested again the whole thing with another character. Nothing odd to report. You can close the issue, for all i am concerned.
